src/main.py

```diff
@@ -284,10 +284,8 @@ def _handle_result(self, result):
             self._play_assistant_response(result.response_audio)
         elif result.transcript:
             logger.warning('%r was not handled', result.transcript)
-            self.say(_("I don’t know how to answer that."))
         else:
             logger.warning('no command recognized')
-            self.say(_("Could you try that again?"))
```

Comment:
For reference, this happens in these cases:
- user tried to give a command, but we failed to recognize any speech (unlikely)
- false positive trigger followed by non-speech audio

The currently behaviour is only useful in the first case, and annoying in the second.